source_gen/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.0.0-wip
+## 2.0.0
```

Comment:
PUT SCREAMING COMMENTS HERE about not publishing until konstantine does all of his work!

Reply:
Done.